### PR TITLE
Fix pack numbers in BOMs

### PR DESCRIPTION
**Pull request by roscopeco**

### CODE REVIEW

File: docs/r2_030_r2.00_bom.pdf
Comment (0xTJ) [inline docs/r2_030_r2.00_bom.pdf:1]: There are some empty lines after the LED4,LED2 row, which don't seem to be used.

Comment (0xTJ) [inline docs/r2_030_r2.00_bom.pdf:1]: The package field for the MC68030RC has MC68020RC.

**Review by 0xTJ — CHANGES_REQUESTED** [review]
Looks good, just one issue and a small note. :smiley:

File: docs/r2_030_r2.00_bom.pdf
Comment (roscopeco) [reply docs/r2_030_r2.00_bom.pdf:1]: Good spot, don't know how I missed that :D 

Thanks, fixed.

Comment (roscopeco) [reply docs/r2_030_r2.00_bom.pdf:1]: Also good spot, thanks again, fixed :)

**Review by 0xTJ — APPROVED** [review]
As far as I can tell, it looks good! :+1:

**Pull request merged**